nanoFramework.Tools.DebugLibrary.Shared/WireProtocol/Engine.cs

```diff
@@ -921,7 +921,6 @@ public void StopProcessing()
                 try
                 {
                     _backgroundProcessor.Join(TimeSpan.FromSeconds(1));
```

Comment:
Why was the `Abort` removed from here?

Reply:
This line with Thread.Abort is the most important change of all! This caused exceptions inside of the catch of try-catch. In Engine.IncomingMessagesListener() the brake commands caused errors. Because of the Abort, the thread is no more alive at these moment.
                catch (ThreadAbortException)
                {
	        --->    break; 
                }
Also, Thread.Abort is marked as Obselete (deprecated) and should no longer be used. And it works very well without that line.

Reply:
Understood. Let's see how this works in the real thing with the VS extension.
If needed we can always revisit here.